frontend/src/components/Auth/Login/Login.js

```diff
@@ -37,7 +21,31 @@ const form = {
 };
 
 const Login = () => {
-  // const history = useHistory();
+  const history = useHistory();
+  const isLogged = useContext(LoggedContext);
+
+  useEffect(() => {
+    if (isLogged) {
+      history.replace("/menu");
+    }
+  }, [isLogged]);
+
+  async function handleSubmit(params, setStatus) {
+    const formData = new FormData();
+    formData.append("username", params.data.username);
+    formData.append("password", params.data.password);
+    (async () => {
+      try {
+        await ky.post("/api/login", {
+          body: formData
+        });
+        Cookies.set("username", params.data.username);
```

Comment:
This should not be handled on frontend, backend should set the cookie and set the domain to your app's domain.